packages/snap/src/utils/caip.ts

```diff
@@ -0,0 +1,67 @@
+import {
```

Comment:
Maybe we could just re-export `CaipNamespaces` and `toCaipChainId` in this file now that we have access to `@metamask/eth-snap-keyring`?

But maybe that's not a really good pattern?

At some points we would have quite a lot of those CAIP helpers, and we could move all of them to `@metamask/utils` but for now, let's keep things simple, this could come in a later refactor. :D